pydis_site/apps/api/viewsets/bot/nomination.py

```diff
@@ -148,10 +160,44 @@ class NominationViewSet(CreateModelMixin, RetrieveModelMixin, ListModelMixin, Ge
     serializer_class = NominationSerializer
     queryset = Nomination.objects.all()
     filter_backends = (DjangoFilterBackend, SearchFilter, OrderingFilter)
-    filter_fields = ('user__id', 'actor__id', 'active')
-    frozen_fields = ('id', 'actor', 'inserted_at', 'user', 'ended_at')
+    filter_fields = ('user__id', 'active')
+    frozen_fields = ('id', 'inserted_at', 'user', 'ended_at')
     frozen_on_create = ('ended_at', 'end_reason', 'active', 'inserted_at')
 
+    def list(self, request: HttpRequest, *args, **kwargs) -> Response:
+        """
+        DRF method for listing Nominations.
+
+        Called by the Django Rest Framework in response to the corresponding HTTP request.
+        """
+        queryset = self.filter_queryset(self.get_queryset())
+        data = NominationSerializer(queryset, many=True).data
+
+        for i, nomination in enumerate(data):
+            entries = NominationEntrySerializer(
+                NominationEntry.objects.filter(nomination_id=nomination["id"]),
+                many=True
+            ).data
+            data[i]["entries"] = entries
+
+        return Response(data)
+
+    def retrieve(self, request: HttpRequest, *args, **kwargs) -> Response:
+        """
+        DRF method for retrieving a Nomination.
+
+        Called by the Django Rest Framework in response to the corresponding HTTP request.
+        """
+        nomination = self.get_object()
+
+        data = NominationSerializer(nomination).data
+        data["entries"] = NominationEntrySerializer(
+            NominationEntry.objects.filter(nomination_id=nomination.id),
+            many=True
+        ).data
+
+        return Response(data)
```

Comment:
My guess is that we don't need this if we make sure to serialize a `Nomination` with its related `NominationEntry` information in the serializer part of the framework. Saves us from overwriting a method.